lib/database_rewinder.rb

```diff
@@ -82,16 +82,28 @@ def all_table_names(connection)
       end
     end
 
+    def get_config_from(connection_name)
+      if Gem::Version.new(Rails.version) >= Gem::Version.new("6.0.0")
+        database_configuration.configs_for(env_name: connection_name).first.configuration_hash
```

Comment:
`configs_for` is available on Rails 6.0 but `configuration_hash` is only available on Rails 6.1. This causes the error on the [build](https://travis-ci.org/github/amatsuda/database_rewinder/jobs/750170204#L396).